### PR TITLE
Move the gmf mobile app outside the examples

### PR DESCRIPTION
**Pull request by elemoine**
This PR adds a `contribs/gmf/apps` directory, and moves the GMF mobile app inside a subdirectory `mobile` of that directory. This PR also changes the structure of the mobile app, to make it close to the structure of an ngeo application in a GMF project.

The PR is still WIP, but you can already run `make serve`, and open http://localhost:3000/contribs/gmf/apps/mobile/index.html in your browser to execute the application in development mode.

You can also run `make build-gmf-mobile-app` to build the application, i.e. generate `build.js` and `build.css` in the `contribs/gmf/apps/mobile/build` directory.

What's missing is the publishing of the mobile app on camptocamp.github.io. More specifically:

* [x] copy the mobile HTML page to `.build/examples-hosted/contribs/gmf/apps/mobile/`
* [x] copy `build.js` and `build.css` to `.build/examples-hosted/contribs/gmf/apps/mobile/build/`
* [x] change the page's `script` tags to use the minified js and css files

### CODE REVIEW

**Comment by elemoine** [comment]
The build failure is related to https://github.com/camptocamp/ngeo/issues/386.

**Comment by sbrunner** [comment]
Looks goof for me, @fredj will also have a look :-)

**Comment by fgravin** [comment]
I have done the 3 tasks mentionned by Eric and rebased the new work done by @ger-benjamin on the mobile app example.
I also had to move the `webfonts` folder to `.build/examples-hosted/contribs/gmf/apps/mobile/`.

As dicussed with @pgiraud i removed the `icons` folder because it is not used by the application (to be disccussed with @ger-benjamin).

One thing i'm not sure about about the `Makefile` is when the target `.build/examples-hosted/%.html: examples/%.html` is triggered. I have the same kind of rule for the mobile app and i'm not sure when it should be launched.

File: contribs/gmf/src/gmf.js [hunk 1]
```diff
@@ -3,6 +3,8 @@
  */
 goog.provide('gmf');
 
+goog.require('ngeo');
+
 
 /** @type {!angular.Module} */
 var gmfModule = angular.module('gmf', ['ngeo']);
```
Comment (fredj) [inline contribs/gmf/src/gmf.js]: ```var gmfModule = angular.module('gmf', [ngeoModule.name]);``` instead ?

**Comment by sbrunner** [comment]
She is trigged when we do a make gh-pages :-)

File: contribs/gmf/src/gmf.js (hunk 1 above)
Comment (fgravin) [inline contribs/gmf/src/gmf.js]: Ok.

**Comment by fgravin** [comment]
I updated the code regarding to your feedback.
I also added the copy of the `mobile/index.html` to `.build/examples-hosted/contribs/gmf/apps/mobile/` in the `gh-pages` target (cf my question above).

I think's it's ready for a merge if your review is Ok.

**Comment by fredj** [comment]
please rebase on master and merge. thanks

**Pull request merged**